ext/reflection/php_reflection.stub.php

```diff
@@ -829,3 +829,18 @@ public function getCallable(): callable {}
 
     public function getTrace(int $options = DEBUG_BACKTRACE_PROVIDE_OBJECT): array {}
 }
+
+/**
+ * @strict-properties
+ * @not-serializable
+ */
+final class ReflectionConstant
```

Comment:
I would expect this to be supported and tested for user constants as well. It would help user to locate where constant was defined, get phpdoc above it etc., like for other reflectionable objects like classes, Closures, ...

Reply:
> I would expect this to be supported

What are you referring to here? Constants don't currently track declaration location, or phpDocs. Adding it is out of scope for this PR.

Reply:
constants defined like `define('CONSTANTA', 'Hello A!');`

Reply:
Added this to the tests, thanks!